wsmux/mux.go

```diff
@@ -0,0 +1,24 @@
+package wsmux
+
+import (
+	"github.com/gorilla/websocket"
+	"time"
+)
+
+type Config struct {
+	RemoteCloseCallback func()
+	AcceptDeadline      time.Time
+	ReadDeadline        time.Time
```

Comment:
I doubt these are config values people want... Instead it's probably better to set a `KeepAliveInterval` as `time.Duration`... which controls how often we do ping/pong...